libs/entry-components/styles/modules/components/buttons/_radio.scss

```diff
@@ -0,0 +1,13 @@
+@use 'sass:map';
+
+@mixin generate-from($theme) {
+	.mat-mdc-radio-button {
+		.mdc-radio {
+			.mdc-radio__native-control:enabled:checked + .mdc-radio__background {
+				.mdc-radio__outer-circle, .mdc-radio__inner-circle {
+					border-color: map.get($theme, 'general', 'buttons', 'radio', 'background') !important;
```

Comment:
@typhoon41  Acknowledging that the use important is discouraged, I am unsatisfied with this approach and will seek a more suitable solution. One possibility is to increase specificity by introducing a custom class in the Formly generated files for radio buttons.